test.js

```diff
@@ -202,6 +203,13 @@ describe('chalk.level', () => {
 		assert.equal(chalk.level, 1);
 		chalk.level = oldLevel;
 	});
+
+	it('should disable colors if they are not supported', () => {
+		return execa('node', ['fixture'])
+			.then(result => {
+				assert.equal(result.stdout, 'test');
+			});
+	});
```

Comment:
It will always fail at https://github.com/chalk/supports-color/blob/master/index.js#L44-L46 when running as a spawned instance because `stdout` in [options.stdio](https://nodejs.org/api/child_process.html#child_process_options_stdio) being `pipe`. `execa('node', ['fixture'], {stdio: ['pipe', 'inherit', 'pipe')` would pass. So yeah, this test is good.